can/interfaces/pcan/pcan.py

```diff
@@ -528,6 +528,32 @@ def _detect_available_configs():
             )
         return channels
 
+    def status_string(self):
+        """
+        Query the PCAN bus status.
+        :rtype: string
+        :return: The status in string.
+        """
+        status = self.status()
+        if status == PCAN_ERROR_OK:
+            return "OK"
+        elif status == PCAN_ERROR_XMTFULL:
+            return "XMTFULL"
+        elif status == PCAN_ERROR_OVERRUN:
+            return "OVERRUN"
+        elif status == PCAN_ERROR_BUSLIGHT:
+            return "BUSLIGHT"
+        elif status == PCAN_ERROR_BUSHEAVY:
+            return "BUSHEAVY"
+        elif status == PCAN_ERROR_BUSWARNING:
+            return "BUSWARNING"
+        elif status == PCAN_ERROR_BUSPASSIVE:
+            return "BUSPASSIVE"
+        elif status == PCAN_ERROR_BUSOFF:
+            return "BUSOFF"
+        else:
+            return "NonImplementedError"
```

Comment:
I' return `None` in this case, to make it more explicit that something went wrong.

Reply:
Yes, good idea. However, when looking back on this, wouldn't it be better if I made a dictonary with all the errors and their strings in `basic.py` and retrieve them with this function, so the class in `pcan.py` doesn't get cluttered much?

Reply:
@felixdivo see commit https://github.com/hardbyte/python-can/pull/725/commits/890128f40da623299f153d7e0fed1de028f2d417